common/app/ab_headlines/ABTestHeadlines.scala

```diff
@@ -0,0 +1,75 @@
+package ab_headlines
+
+import java.net.URI
+
+import com.google.gdata.client.spreadsheet.{FeedURLFactory, SpreadsheetService}
+import com.google.gdata.data.spreadsheet.{WorksheetFeed, ListFeed}
+import common.{Logging, AutoRefresh}
+import conf.{Switches, Configuration}
+import play.api.libs.json.Json
+
+import scala.collection.JavaConverters._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.blocking
+import scala.concurrent.duration._
+
+case class HeadlineABTest(
+  testId: Int,
+  headlines: Set[String]
+)
+
+object ABTestHeadlines extends AutoRefresh[Map[String, HeadlineABTest]](0.seconds, 15.seconds) with Logging {
```

Comment:
I think the answer to the original question is no, unless the headlines are going to be rolled quite aggressively?